src/main/java/nl/tudelft/contextproject/script/UpdateTask.java

```diff
@@ -0,0 +1,20 @@
+package nl.tudelft.contextproject.script;
+
+import nl.tudelft.contextproject.ContextTFP;
+
+import java.util.TimerTask;
+
+public class UpdateTask extends TimerTask {
```

Comment:
I don't really mind to have an inner class. UpdateTask seems so trivial to me that it doesn't deserve a separate class.